expcore/Mini_games.lua

```diff
@@ -96,16 +101,19 @@ function Mini_games._prototype:add_event(event_name,func)
 end
 
 
-function Mini_games.start_game(name,...)
-
-    local parse_args = {...}
+function Mini_games.start_game(name,parse_args)
     local mini_game = Mini_games.mini_games[name]
     if mini_game == nil then
         return "This mini_game does not exsit"
     end
-
-    if  mini_game.options ~= #parse_args then
-        return "Wrong number of arguments"
+    if parse_args then
+        if  mini_game.options ~= #parse_args then
+            return "Wrong number of arguments"
+        end
+    else 
+        if mini_game.options ~= 0 then
+            return "Wrong number of arguments"
+        end
     end
```

Comment:
Why do you return a string error message? Should you not be using error or assert to check and raise an error when you get unexpected arguments.

Reply:
i return the string for when /start is used as this will print it out as an error